### PR TITLE
Support for Vue 3

### PR DESCRIPTION
**Pull request by nstuyvesant**
### Updates
- Updated packages/vue/package.json to have dist-tag of `next`
- Updated README.md to explain support is for Vue 3 and if an older version is needed, use `@carbon/charts-vue@latest`
- Maintenance bump for Storybook and tslib (within existing semver)
- Updated demo scripts to use StackBlitz EngineBlock template for Vue 3 instead of generic node (starts faster and doesn't require devDependencies)
- Converted components to use a factory based on Vue Composition API. All components are now 3 lines long.
- Updated Storybook to use Vue Composition API.

NOTE: Unable to do final testing in StackBlitz since there is no published `@carbon/charts-vue` component library for Vue 3; however, it tests out in a local Vue 3 project and in Storybook locally.

### Review checklist (for reviewers only)
- [ ] Demos all features
- [ ] Documented/annotated
- [ ] Matches UI/UX specs
- [ ] Meets the [code style guide](https://github.com/carbon-design-system/carbon-charts/wiki/Code-style-guide)
- [ ] Accessible
- [ ] Mobile first (responsive)
- [ ] RTL support (bidirectional text)
- [ ] Performant (limited bloat)

### CODE REVIEW

File: packages/vue/src/components/CcvChoroplethChart.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { ExperimentalChoroplethChart as ChoroplethChart, type ChoroplethChartOptions } from '@carbon/charts'
+import { chartFactory } from './chartFactory'
+
+export default chartFactory<ChoroplethChartOptions>(ChoroplethChart, 'CcvChoroplethChart')
```
Comment (theiliad) [inline packages/vue/src/components/CcvChoroplethChart.ts:4]: ExperimentalCcvChoroplethChart

**Review by theiliad — CHANGES_REQUESTED** [review]

File: packages/vue/src/components/CcvChoroplethChart.ts (hunk 1 above)
Comment (nstuyvesant) [reply packages/vue/src/components/CcvChoroplethChart.ts]: Just pushed the requested change.

**Review by theiliad — APPROVED** [review]
LGTM

**Comment by raizdev** [comment]
@theiliad Im experience some issues, could take a look? https://github.com/carbon-design-system/carbon-charts/discussions/1641

**Pull request merged**